stock_move_auto_assign/tests/test_auto_assign.py

```diff
@@ -2,7 +2,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 from odoo.addons.queue_job.job import identity_exact
-from odoo.addons.queue_job.tests.common import mock_with_delay
+from odoo.addons.queue_job.tests.common import mock_with_delay, trap_jobs
```

Comment:
Could you please remove the deprecated mock_with_delay
I created a PR https://github.com/camptocamp/stock-logistics-warehouse/pull/2

Reply:
I updated/fix the tests. no more with_delay

Reply:
thx